R/regression.R

```diff
@@ -62,6 +64,10 @@
 #' @param recursive.call Used internally to indicate if call is a result of recursion (e.g., multiple imputation).
 #' @param effects.format A list of items \code{max.label} (the maximum length of a factor label on the x-axis)
 #'   and \code{y.title} (the title of the y-axis, defaults to outcome label).
+#' @param outlier.prop.to.remove A single numeric value that determines the percentage of data points to remove from the
+#'   analysis. The data points removed correspond to those in the proportion with the largest residuals.
+#'   A value of 0 or NULL would denote no points are removed. A value x, with 0 < x < 0.5 (not inclusive) would
+#'   denote that a percentage between none and 50\% of the data points are removed.
 #' @param ... Additional argments to be passed to  \code{\link{lm}} or, if the
 #'  data is weighted,  \code{\link[survey]{svyglm}} or \code{\link[survey]{svyolr}}.
 #' @details In the case of Ordered Logistic regression, this function computes a proporional odds model using
```

Comment:
Would be good to document how the outlier detection works in the details and discussed the types of residuals used in the various cases.

Reply:
Added more details in the documentation. Let me know if more detailed are required.